pydicom/util/codify.py

```diff
@@ -171,9 +171,9 @@ def code_sequence(dataelem,
         #    use that; if one with 'Number', use that, else start at 1
         index_keyword = seq_keyword.replace("Sequence", "") + "Index"
         number_keyword = seq_keyword.replace("Sequence", "") + "Number"
-        if index_keyword in ds:
+        if hasattr(ds, index_keyword):
             index_str = str(getattr(ds, index_keyword))
```

Comment:
Why the switch to `hasattr` for these?

Reply:
Because it was trying keywords weren't valid, one was something like `DoseReferenceInstance`